libsel4/tools/syscall_stub_gen.py

```diff
@@ -37,1012 +37,29 @@
 #     XML method descriptions are added.
 #
 
-import operator
-import itertools
-import xml.dom.minidom
 from argparse import ArgumentParser
 import sys
-from functools import reduce
+from generator import WORD_SIZE_BITS_ARCH
```

Comment:
Do we need this import here?

Reply:
Ditto as above, I've created `Architecture` class aiming to solve this.